frontend/appflowy_flutter/lib/plugins/document/presentation/plugins/emoji_picker/emoji_menu_item.dart

```diff
@@ -4,6 +4,79 @@ import 'package:flutter/services.dart';
 
 import 'emoji_picker.dart';
 
+ShortcutEvent showEmojiPickerEvent(BuildContext context) => ShortcutEvent(
+      key: 'Show emoji picker',
+      command: 'meta+e',
+      windowsCommand: 'ctrl+e',
+      linuxCommand: 'ctrl+e',
+      macOSCommand: 'cmd+e',
```

Comment:
cc @annieappflowy you didn't specify a key combination for this shortcut in https://github.com/AppFlowy-IO/AppFlowy/issues/2038. Are these correct?

Reply:
We can remove the` macOSCommand: 'cmd+e'` line since `command:'meta+e'` will take care of the Mac OS Command.

Reply:
I prefer the other way around,
```suggestion
      command: 'ctrl+e',
      macOSCommand: 'meta+e',
```